### PR TITLE
Added Pull Request

### PR DESCRIPTION
**Pull request by niteshkumarniranjan**
Completed my calculator.

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks! Please provide a link to a test setup, where we can see the app in action.

**Comment by niteshkumarniranjan** [comment]
[http://niteshkumarniranjan.me/ember-calc-host/](http://niteshkumarniranjan.me/ember-calc-host/)

**Comment by poush** [comment]
Great work @niteshkumarniranjan. There is one feature you can add is the ability to display more than 9 characters on screen. Currently, if you calculator gets more than 9 characters on screen then i doesn't display any more but yeah it still works. eg 999999999-20

One thing you can do is reduce the text size or make it scrollable on more than 9 characters.

File: niteshkumarniranjan/embercalculator/app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	float: left;
+}
+
+.top .screen {
+  height: 130px;
+  width: 100%;
+	float: right;
+	padding: 0 10px;
+	background: #e3e7ea;
+  font-size: 52px;
```
Comment (poush) [inline niteshkumarniranjan/embercalculator/app/styles/app.css]: Be consistent with your indentations. Don't add unnecessary indentations.

Comment (niteshkumarniranjan) [reply niteshkumarniranjan/embercalculator/app/styles/app.css]: Okay!

**Comment by niteshkumarniranjan** [comment]
@poush I have worked on your ideas. Thank you for suggesting me.

**Comment by vaibhavsingh97** [comment]
@niteshkumarniranjan Have you fixed this PR as sugested by the @poush

**Comment by niteshkumarniranjan** [comment]
Yeah!

**Comment by vaibhavsingh97** [comment]
The changes are not refleted to the deployed link

**Comment by niteshkumarniranjan** [comment]
You can check it [http://niteshkumarniranjan.me/ember-calc-host/](http://niteshkumarniranjan.me/ember-calc-host/)

**Comment by niteshkumarniranjan** [comment]
When it text is more than 9 chars it adds a scroll bar to display.

**Comment by niteshkumarniranjan** [comment]
@vaibhavsingh97 Is there anything left to do.

**Pull request merged**